recipes/spdlog_setup/all/conandata.yml

```diff
@@ -0,0 +1,5 @@
+sources:
+  "1.1.0":
+    url:
+      - "https://github.com/gegles/spdlog_setup/archive/v1.1.0.tar.gz"
```

Comment:
We generally do not accept forks  especially from active projects. It's very difficult to see what changes you made since you renamed files? Maybe?

It there a particular reason to not use the official repository? Gambines code is usually pretty good and accepts contributions

Reply:
I hear you, but I don't believe this is actively maintained at all... That's why I had to fork and revamp the project... 

BTW, unless I am mistaken, the maintainer is not [gabime](https://github.com/gabime) (the creator of `spdlog`), but [guangie88](https://github.com/guangie88)... 

I can try to reach again and see what can be done.. but the project I now maintain has changed quite a bit from his...

Reply:
@guangie88 Any thoughts? Would you be open to either take my changes? or transfer ownership of the lib? Let me know. thx!

Reply:
Ahh I didn't notice the difference last night! Thanks for pointing that out:)

If that's the case, we can add the author prefix so people know it's a fork :)

`author-name` is the special way

Reply:
That's totally fine with me, happy to refactor anything. I am assuming you're just asking to rename the recipe. Let me update the PR ASAP. thx!

Reply:
Ok @prince-chrismc, I've now renamed the recipe to `gegles-spdlog_setup`.